pkg/controller/container-runtime-config/container_runtime_config_bootstrap.go

```diff
@@ -33,7 +33,7 @@ func RunContainerRuntimeBootstrap(templateDir string, crconfigs []*mcfgv1.Contai
 			}
 			role := pool.Name
 			// Generate the original ContainerRuntimeConfig
-			originalStorageIgn, _, _, err := generateOriginalContainerRuntimeConfigs(templateDir, controllerConfig, role, featureGateAccess)
```

Comment:
verify is complaining that the function header still has featuregate being passed in